tools/backups2datalad/__main__.py

```diff
@@ -3,7 +3,6 @@
 import logging
```

Comment:
could there be some rudimentary test, even if smoke one to start with to import this module?

Reply:
Done.